packages/_vue3-migration-test/package.json

```diff
@@ -16,7 +16,7 @@
     "@vueuse/core": "~10.7.1",
     "vue": "^3.4.22",
     "vue-router": "^4.3.0",
-    "vuex": "^4.1.0"
+    "vuex": "4.0.2"
```

Comment:
Reactivity of vuex getters is broken in `4.1.0` when registering store modules dynamically. https://github.com/vuejs/vuex/issues/2217
It caused emitters didn't work when the trigger is a getter and not the state (`request` getter of QuerySuggestions x-module).
Downgraded to `4.0.2`